code-of-conduct.md

```diff
@@ -85,7 +85,6 @@ public domain, no credit and no open licensing of your version is required.
 * Please see the [Additional Resources][add-rec] document
 
 [organizers]: conduct-organizers.md
-[add-rec]: additional-resources.md
```

Comment:
I saw this was a dead link, but could you 1) retain the link and 2) update this to conduct-resources.md? Thanks